Task 2/Task 2.1/Task 2.1/firure/Square.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Task_2._1
+{
+    public class Square: Rectangle, IAeraing
+    {
+        public Square(Position pos, double side) : base(pos, side, side) { }
+    }
```

Comment:
From development standpoint it's better to have a rectangle that can say that it's a square - square is a rectangle anyway. Though this inheritance is okay for your case, if rectangle class changes so it can change its size, square will change too - this way you will be able to change Square to have different sides, which is unacceptable.